test/extended/testdata/router/router-shard.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+parameters:
+- name: NAME
+- name: DOMAIN
+- name: NAMESPACE
+- name: TYPE
+objects:
+- apiVersion: operator.openshift.io/v1
+  kind: IngressController
+  metadata:
+    name: ${TYPE}
+    namespace: ${NAMESPACE}
+    annotations:
+      ingress.operator.openshift.io/default-enable-http2: "true"
+  spec:
+    replicas: 1
+    domain: ${DOMAIN}
+    endpointPublishingStrategy:
+      type: LoadBalancerService
```

Comment:
This test should probably only ever run on platforms that support LoadBalancerService, right? We may need to include skips for all other platforms at the top of the http/2 test.

Reply:
Done in a46c9a5